src/components/shared/CoinIcons/DynamicCoin.tsx

```diff
@@ -4,7 +4,7 @@ import { shortSymbol } from '../../../helpers/tokenHelpers'
 import { DynamicCoinWrapper } from './DynamicCoin.style'
```

Comment:
Could you rename folder `CoinIcons` to `icons` or so? Just to have folder names in lowercase.